### DISCUSSION TITLE
Submission Error 400

### DISCUSSION THREAD

**Discussion by ismailatitallah** [OP-TED/eForms-SDK#287] [Q&A] [answered]
Hello, I am trying to submit an xml. When I validate the xml, all is OK. Schematron validate my xml. But when I submit the xml to enotice, I have 400 without any message.
curl -X 'POST' \
  'https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/submit' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer xxxxxxxxxx' \
  -H 'Content-Type: multipart/form-data' \
  -F 'notice=@notice-508127-16-8123041d-aa9d-466f-a516-2b5915e448e6.xml;type=text/xml' \
  -F 'metadata={
  "noticeAuthorEmail": "john@doe.com",
  "noticeAuthorLocale": "en"
}'
Does anyone have a hint to help me? please? (the validation report SVRL + the notice XML are in the files.zip)
[files.zip](https://github.com/OP-TED/eForms-SDK/files/10294490/files.zip)

**Comment by rouschr** [comment, marked as answer]
Hello,
note that eNotices2 is developed separately, for issues or questions concerning this project use this form:
https://enotices2.ted.europa.eu/help/contact

Note that you can use the CVS API to perform validation of a notice:

https://docs.ted.europa.eu/api/index.html
https://docs.ted.europa.eu/api/endpoints/cvs-ted-europa-eu.html

**Comment by pcolast** [comment]
Dear ismailatitallah,

I tried to validate your notice using cvs in preview and it seems it is not valid.
It seems there is an issue with the order of elements.
Sequence is validated with the XSD (i.e. the schema). Schematron will check for existence, content, number of occurences, but not for the order.

Best regards,
Stefan
<img width="899" alt="q-287" src="https://user-images.githubusercontent.com/117370985/211576620-610b77df-b7ab-499b-9c0f-f0cbfe35703c.PNG">
